### DISCUSSION TITLE
How to install and run this software on Windows 11?

### DISCUSSION THREAD

**Discussion by saadbhattii** [fra589/cn5X#43] [Q&A] [answered]
Hello, I have a 5-Axis CNC machine, I am currently using GRBL-Gru as a g-code sender and I wanted to try using this software as a G-Coder sender instead, I downloaded the zip file but I am not unaware on how to install it on my windows 11, can you guide me through it?

Thankyou.

**Comment by fra589** [comment, marked as answer]
Hello @saadbhattii,  
  
Install documentation is on the cn5X++ wiki page : https://github.com/fra589/cn5X/wiki/Install-cn5X-on-Windows

This has been tested only on Windows 10 since I work on Linux and I don't have Windows 11 for testing. But it should work on W11.
Don't hesitate to tell me if you have specific problem on Windows 11.

Currently, I'm working on migrate the application from QT5 to QT6. 
The code migration is nearly finished, it just need to do full functional tests, this may take few time.
But this should be done shortly... Stay connected :-)

@++;
Gauthier.

**Reply by saadbhattii** [reply]
Thank you for the help!
